### DISCUSSION TITLE
2023/09/07/luban-platform-in-actions-monorepo-and-rspack/index

### DISCUSSION THREAD

**Discussion by giscus** [sumy7/sumy7.github.io#7] [Announcements]
# 2023/09/07/luban-platform-in-actions-monorepo-and-rspack/index

SumyGG's Blog

https://sumygg.com/2023/09/07/luban-platform-in-actions-monorepo-and-rspack/index.html

**Comment by yizixu** [comment]
非常好的文章，可以参考下 luban-icons 的代码实现以及图标使用方式吗，感激不尽

**Reply by sumy7** [reply]
hi, 感谢关注。
我制作了一个简单的demo，包括一些打包和构建指令。你可以看一下如何将代码引用到你的memorepo里。
https://github.com/sumy7/icon-pioneer
